### ISSUE TITLE
Lightcurve data doesn't seem correct compared to plots on lasair.

### ISSUE THREAD

**Issue by dwright04** [rafia17/Zooniverse_SLSN#12]
The first subject in this subject set:

https://www.zooniverse.org/lab/10996/subject-sets/81017

The metadata for this subject (subject_id=40997100) says the object_id=ZTF19acykqdk.  Looking at the lasair page for this object (https://lasair-dev.roe.ac.uk/object/ZTF19acykqdk/) the lightcurves do not look at all similar and I don't understand why.  The x-axis range is very similar.

The magnitudes do not seem to correspond.  The lasair page has no detections brighter than 18th but the subject plot does.

@rafia17 can you look into this?

It seems that either the wrong object_id is associated with the lightcurve data or the plotting/data parsing isn't correct.

**Comment by rafia17** [comment]
Yes, I also noticed this and had in my mind to debug the parsing algorithm once I am done with populating the subject sets. The object_Id is fine, but there is definitely something going on with the parsing method that I will look into soon.
